### ISSUE TITLE
A question about test data

### ISSUE THREAD

**Issue by Wangx-spec** [GIST-CSBL/BayeshERG#3]
Hello, I have already cloned and run the code. 
I noticed that the accuracy of test-all file in your paper is about 0.75 (in Figure 2), but my result is 0.87(after run the model and calculated data in the output file), which is higher than the result in paper. The same problem also happened in test_rev data. However, results in external data (EX1~EX4) showed no problem.
I don't know what happened. Could you please provide codes that used to calculate the accuracy, if it is permitted?
Thank you so much.

**Comment by sunset222** [comment]
Hi, 
Thank you for your interest in my work. I am sorry for the late reply about your issue.
The final published model weights were trained with the whole fine-tuning dataset. However, in the internal validation section of the paper, we tested the performance with the model trained without the validation and test sets. Thus, those test data were seen during training, so it shows higher performance than the internal test results. 

If you have any further questions, please don't hesitate to contact me. Thank you again.
